graphql2/schema.graphql

```diff
@@ -293,6 +303,16 @@ input CreateScheduleInput {
   targets: [ScheduleTargetInput!]
 }
 
+input CreateCalendarSubscriptionInput {
+  name: String!
```

Comment:
This type is missing the `ScheduleID` and the `NotificationMinutes` array.
https://github.com/target/goalert/compare/calendar-subscription-api#diff-04e3af9ffc876a383b6f192a8ff8450dR269

We should also probably allow passing in `disabled` to keep the API matching between create and update (i.e. update is a subset of create)